### DISCUSSION TITLE
how to use multiple value in artist metadata field

### DISCUSSION THREAD

**Discussion by Kri-U** [fooyin/fooyin#294] [Q&A] [answered]
I have several tracks that have multiple artists involved (for example a Remix or a Featuring), from my experience in Musicbee, I was entering `artist1; artist2` in the Artist metadata field and Musicbee would show the track under both of theses artist, but here Fooyin just show `artist1; artist2` as a whole artist. Is there a way to have multiple value in the artists Metadata field in Fooyin ?
I have an example of my problem below :
![image](https://github.com/user-attachments/assets/68832331-e6f7-4bdf-bd06-b7728e6e8245)

**Comment by ludouzi** [comment, marked as answer]
The default artist column for the library filters (`%<artist>%`) automatically splits multiple values into separate rows. 

I'm guessing the reason it's not doing so for the above is that the artists were written to a single tag entry, and so fooyin reads it as a single artist. You may be able to confirm that with a tag editor like kid3.

A possible solution is to split on `;` when reading, but I'd rather avoid that as it's a valid character and may lead to incorrect splits in some cases.

**Reply by Kri-U** [reply]
Thanks, I'm willing to split at `;` when reading as I'm using this character only for artist separation and I don't want to update all my metadata yet. How can I achieve this ?

**Reply by ludouzi** [reply]
I've just added a $split function with https://github.com/fooyin/fooyin/commit/521a2992657094039ddaec3db8470404d4a957cd. 

If you were to add a custom column with `$split(%<artist>%,; )`, it should split artists like the above into multiple fields.

**Reply by Kri-U** [reply]
thanks a lot !
